modules/renter.go

```diff
@@ -67,9 +67,20 @@ type DownloadInfo struct {
 	StartTime   time.Time `json:"starttime"`
 }
 
+// An Allowance dictates how much the Renter is allowed to spend in a given
+// period.
+type Allowance struct {
+	Funds  types.Currency    `json:"funds"`
+	Hosts  int               `json:"hosts"`
+	Period types.BlockHeight `json:"period"`
+}
```

Comment:
yeah. Currently I use the `renewThreshold` helper function for that, but it's better to be explicit.